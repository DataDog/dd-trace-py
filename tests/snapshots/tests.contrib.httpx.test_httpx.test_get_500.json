[[
  {
    "name": "http.request",
    "service": null,
    "resource": "http.request",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "http",
    "error": 1,
    "meta": {
      "http.method": "GET",
      "http.status_code": "500",
      "http.url": "http://localhost:8001/status/500",
      "runtime-id": "0c15188ec3454af48be672464b951d64"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 85259
    },
    "duration": 5397000,
    "start": 1633032093706602000
  }]]
