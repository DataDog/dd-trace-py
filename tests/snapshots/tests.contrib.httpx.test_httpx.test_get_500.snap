[[{"name" "http.request"
   "service" nil
   "resource" "http.request"
   "type" "http"
   "error" 1
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1620689782699314000
   "duration" 3836000
   "meta" {"runtime-id" "0975fd0faf2a4855b8e668f3a21af4d4"
           "http.method" "GET"
           "http.url" "http://localhost:8001/status/500"
           "http.status_code" "500"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 4674
              "_dd.tracer_kr" 1.0}}]]
